src/ng/filter/limitTo.js

```diff
@@ -96,12 +96,12 @@ function limitToFilter() {
     } else {
       limit = toInt(limit);
     }
-    if (isNaN(limit)) return input;
+    if (isNumberNaN(limit)) return input;
 
     if (isNumber(input)) input = input.toString();
     if (!isArray(input) && !isString(input)) return input;
 
-    begin = (!begin || isNaN(begin)) ? 0 : toInt(begin);
+    begin = (!begin || isNumberNaN(begin)) ? 0 : toInt(begin);
```

Comment:
I don't think this one should change. This one specifically wanted `"123"` to pass...